tests/content-types.js

```diff
@@ -3,20 +3,38 @@ import config from 'config';
 import types from '../lib/content-types';
 import util from 'util';
 import fs from 'fs';
+import only from '../lib/content-types/only.js';
+var bar_input = require('./fixtures/objects//bar_input.js')
```

Comment:
Please use the same import statements as above. This is something linting should have caught and didn't. 

I'm going to need to go in and make sure our linter is working correctly before we can merge this as I'm seeing lots of linting issues in your code